Makefile

```diff
@@ -51,6 +51,19 @@ ifeq ($(aws_region), us-gov-west-1)
 	source_ami_owners ?= 045324592363
 endif
 
+# The following _FLAG vars are to be passed into hack/latest-binaries.sh 
+ifdef aws_profile
+	PROFILE_FLAG := --profile $(aws_profile)
+endif
+
+ifdef binary_bucket_name
+	BUCKET_FLAG := --bucket $(binary_bucket_name)
+endif
+
+ifdef binary_bucket_region
+	REGION_FLAG := --region $(binary_bucket_region)
+endif
+
```

Comment:
I think this should do what you want:
```suggestion
# make available in hack/latest-binaries.sh 
export binary_bucket_region
export binary_bucket_name
```

Reply:
It looks like the exported variables are not being picked up by$(shell ....)